atorch/atorch/utils/loss_spike_utils.py

```diff
@@ -0,0 +1,146 @@
+""" loss spike utils
+Save loss spike to files;
+Decode loss spike and save Corresponding sample to a file.
+"""
+import os
+import datetime
+
+import numpy as np
+from atorch.common.log_utils import default_logger as logger
+
+
+class LossSpikeBase:
+    def __init__(self, loss_spike_save_dir, sample_data_paths, each_sample_len, min_iter, min_loss,
+                 loss_info_splitter='\t', loss_sample_str_splitter=','):
+        """
+        init params
+        Args:
+            loss_spike_save_dir: str, The directory where loss spike files are stored
+            sample_data_paths: 样本存储的路径信息, 可以是用户自定义的结构，如名称+路径的tuple list:
```

Comment:
It is better to use English annotations.

Reply:
Done.